src/Login/components/LoginWithOptions/LoginWithOptions.js

```diff
@@ -14,7 +14,7 @@ import LoginWithMnemonic from '../../components/LoginWithMnemonic/LoginWithMnemo
 import LoginWithPrivateKey from '../../components/LoginWithPrivateKey/LoginWithPrivateKey'
 import LoginTrezor from '../../components/LoginWithTrezor/LoginWithTrezor'
 import LoginWithWallet from '../../components/LoginWithWallet/LoginWithWallet'
-import { bccProvider, btcProvider } from '../../network/BitcoinProvider'
+import { bccProvider, btcProvider, btgProvider, ltcProvider } from '../../network/BitcoinProvider'
```

Comment:
split folder 'network' to appropriate blockchains, ie `network/bitcoin/*`, `network/ethreum/`, etc.

Reply:
Will split in the future.
Right now better to accept this PR to prevent troubles (Andrey moved network folder to another location).